libs/application/templates/parental-leave/src/forms/ParentalLeaveForm.ts

```diff
@@ -731,10 +730,10 @@ export const ParentalLeaveForm: Form = buildForm({
             }),
           ],
         }),
-        buildDescriptionField({
+        buildCustomField({
           id: 'thankYou',
           title: mm.finalScreen.title,
-          description: mm.finalScreen.description,
+          component: 'Conclusion',
```

Comment:
It would be nice to refactor out "magic" constants, custom components could be referenced through an enum defined in constants.ts

```typescript
// constants.ts
enum CustomComponents {
  Conclusion = 'Conclusion',
}

// ParentalLeaveForm.ts
buildCustomField({
  // ...
  component: CustomComponents.Conclusion
})
```

Reply:
Agree on this, I'll tackle in separate PR.